app/components/utility/filter_component.html.erb

```diff
@@ -26,7 +26,7 @@
               <ul class="moj-filter-tags">
                 <% tags_for_active_filter(active_filter).each do |tag| %>
                   <li>
-                    <%= link_to(tag[:remove_link], class: 'app-checkbox-filter__tag', id: "tag-#{tag[:title].parameterize}") do %>
+                    <%= govuk_link_to(tag[:remove_link], class: 'app-checkbox-filter__tag', id: "tag-#{tag[:title].parameterize}") do %>
```

Comment:
This file was previously ignored in the config. If we're fixing the ignored error, can we remove the relevant ignore lines? Not sure why we explicitly ignored it though – maybe it's supposed to be a non-govuk link?

Reply:
Good spot—have just updated/cleaned up the ignores for now.